docker-unified/Dockerfile

```diff
@@ -274,6 +274,36 @@ RUN chmod +x /usr/local/bin/hbs
 COPY docker-unified/entrypoint.sh /entrypoint.sh
 RUN chmod +x /entrypoint.sh
 
+#Set up user and group
+RUN addgroup --gid 1000 bitwarden
```

Comment:
We also need to consider the impact of hard-coding the user and group ID in the image. Currently, since they're set at runtime, it's possible to provide custom IDs. Setting them in the Dockerfile removes this capability, and could potentially impact other deployments.